engine/swarm/admin_guide.md

```diff
@@ -251,8 +237,51 @@ Before you forcefully remove a manager node, you must first demote it to the
 worker role. Make sure that you always have an odd number of manager nodes if
 you demote or remove a manager
 
+## Back up the swarm
+
+Docker manager nodes store the swarm state and manager logs in the
+`/var/lib/docker/swarm/raft` directory.
+
+Back up the `raft` data directory often so that you can use it in case of
+[disaster recovery](#recover-from-disaster). To restore, see
+[Restore from a backup](#restore-from-a-backup).
+
+### Things to know before backing up
+
+#### Hot or cold backups
+
+To do a _cold backup_, completely shut down the swarm before backing it up.
+Generally, _hot backups_ are sufficient. However, if any files are being changed
+while the copy is occurring, you may lose Some data. Be sure to test backup
+and recovery  thoroughly before settling on a backup strategy.
+
+#### Which manager to back up from
+
+In Raft, the leader can change at any time. In theory the followers may fall
+behind the leader, so a backup from a follower may miss some of the very latest
+data. However, this should not be an issue. Depending on your data integrity
+needs, you may choose to back up from multiple masters and test the restore to
+be sure that all data is accounted for.
+
 ## Recover from disaster
 
+### Restore from a backup
+
+If you have backed up the Raft directory as described in
+[Backing up the swarm](#backing-up-the-swarm), you can restore the data to a new
+swarm.
+
+1.  Initialize a new single-node swarm.
+2.  Shut down Docker on that swarm.
+3.  Restore the `/var/lib/docker/swarm/raft` directory with the contents of the
+    backup.
+4.  Verify the contents of the backup.
```

Comment:
What I had in mind is that they should figure out their own strategy to verify the contents of the backup, probably based on the application they are running (like an integration test of sorts). In ops land, verifying that restores work is an important step.